grpcclient/grpcclient.go

```diff
@@ -88,6 +98,19 @@ func (cfg *Config) DialOption(unaryClientInterceptors []grpc.UnaryClientIntercep
 		unaryClientInterceptors = append([]grpc.UnaryClientInterceptor{NewRateLimiter(cfg)}, unaryClientInterceptors...)
 	}
 
+	if cfg.ConnectTimeout > 0 {
+		opts = append(
+			opts,
+			grpc.WithConnectParams(grpc.ConnectParams{
+				Backoff: grpcbackoff.Config{
```

Comment:
Should this be built on top of `DefaultConfig`, so that we use the default values for the other properties like `Jitter` and `Multiplier`?

If not, adding a comment that we want to set these to 0 might be a good idea.

Reply:
Great find. I did the following:
- Reused the `grpcbackoff.DefaultConfig`
- If BaseDelay or MaxDelay were explicitly set, use them. Otherwise, use the default.

I decided to do it this way to make this retrocompatible. WDYT?

Reply:
Sounds good to me. Might be worth adding a test or two as well to make sure this doesn't break in the future.

Reply:
I also sent on your DM, but sharing here for visibility :smile: 
I found it quite hard to testing this in a decent way because the []Opts doesn't expose a way for me to recover the final backoff params. Do you have a suggestion on how I can test this? I could write a test just to check if `config.Timeout` was the expected value but tbh that doesn't seem like a valuable test. However, If I could recover the gRPC opt that will be used by the connection the test would be pretty good.